### PR TITLE
[MI][HOSS-192] add opens new tab to internal error page link for acce…

### PR DESCRIPTION
**Pull request by muhammadibraaheem**
…ssibility

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -76,7 +76,7 @@ status-check-failure.heading2CustomerDetails=Customer details
 status-check-failure.listParagraph=This may be for a number of reasons, including:
 status-check-failure.list-item1=The details are incorrect
 status-check-failure.list-item2=The customer has not applied to the EU Settlement Scheme
-status-check-failure.contact-helpdesk-link=IT help desk
+status-check-failure.contact-helpdesk-link=IT help desk(opens link in new tab)
```
Comment (udinobi) [inline conf/messages:79]: Maybe `... desk (opens...`
Is a blank maybe missing between **desk** and the bracket?

Comment (russellthorn) [reply conf/messages:79]: Yes there needs to be a space between desk and (

File: conf/messages [hunk 2]
```diff
@@ -76,7 +76,7 @@ status-check-failure.heading2CustomerDetails=Customer details
 status-check-failure.listParagraph=This may be for a number of reasons, including:
 status-check-failure.list-item1=The details are incorrect
 status-check-failure.list-item2=The customer has not applied to the EU Settlement Scheme
-status-check-failure.contact-helpdesk-link=IT help desk(opens link in new tab)
+status-check-failure.contact-helpdesk-link=IT help desk (opens link in new tab)
```
Comment (russellthorn) [inline conf/messages:79]: 👍

Comment (russellthorn) [inline conf/messages:79]: the story says the link should read "contact the IT help desk (opens in a new tab)"

**Review by russellthorn — COMMENTED** [review]
the story says the link should read "contact the IT help desk (opens in a new tab)"

https://jira.tools.tax.service.gov.uk/browse/HOSS-192?focusedCommentId=1127066&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1127066

https://jira.tools.tax.service.gov.uk/browse/HOSS-192?focusedCommentId=1139754&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1139754

File: conf/messages (hunk 2 above)
Comment (muhammadibraaheem) [reply conf/messages:79]: yes it does, i must  have misread it

File: conf/messages [hunk 3]
```diff
@@ -76,7 +76,7 @@ status-check-failure.heading2CustomerDetails=Customer details
 status-check-failure.listParagraph=This may be for a number of reasons, including:
 status-check-failure.list-item1=The details are incorrect
 status-check-failure.list-item2=The customer has not applied to the EU Settlement Scheme
-status-check-failure.contact-helpdesk-link=IT help desk (opens link in new tab)
+status-check-failure.contact-helpdesk-link=IT help desk (opens in new tab)
```
Comment (russellthorn) [inline conf/messages:79]: the story says the link should read **"contact the IT help desk (opens in a new tab)"**

https://jira.tools.tax.service.gov.uk/browse/HOSS-192?focusedCommentId=1127066&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1127066

https://jira.tools.tax.service.gov.uk/browse/HOSS-192?focusedCommentId=1139754&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1139754

Comment (russellthorn) [reply conf/messages:79]: **"contact the IT help desk (opens in a new tab)"**

**Review by russellthorn — COMMENTED** [review]
"contact the IT help desk (opens in a new tab)"

**Pull request closed without merging**